Marlin/Marlin_main.cpp

```diff
@@ -347,20 +347,31 @@ bool target_direction;
 #endif
 
 #ifdef DELTA
+
+  #define TOWER_1 X_AXIS
+  #define TOWER_2 Y_AXIS
+  #define TOWER_3 Z_AXIS
```

Comment:
It's confusing because it is wrong! There is no X_AXIS associated with TOWER_1. There is an actuator position which we might call the A_AXIS.

Look at the physical layout.  Generally, the printer is viewed from the front with a tower on the left, a tower on the right, and one centered in the back.

Thus, the towers are (nominally) at 210°, 330°, and 90° relative to an XY coordinate system.
And because the builds are not tied to any absolute coordinate system, we can define the build space so that those angles are exact when measured in the coordinate system of the build.

If we want to be explicitly clear, in defining things, we should use terms that do not assume any particular convention for the naming of an axis.
Thus LEFT_TOWER_RADIUS_TRIM might be a term that is more user friendly.

The use of
{LEFT,RIGHT,REAR}_TOWER_{RADIUS,ARM,ENDSTOP}_{DESIGN,ADJUSTMENT,TRIM}
would be clear to the user. We can then use #defines to relate that terminology to actuators that are identified by a letter (ABC) or a digit (0123).